js/apps/system/_admin/aardvark/APP/react/src/views/query/useQueryEditorHandlers.tsx

```diff
@@ -35,6 +36,12 @@ export const useQueryEditorHandlers = () => {
         acc[bindParam] = prevQueryBindParam;
         return acc;
       }, {} as { [key: string]: string });
+
+      const needsUpdate = isEqual(parsedBindParamsMap, prevQueryBindParams);
+      if (needsUpdate) {
+        return prevQueryBindParams;
+      }
```

Comment:
shouldn't it need an update when it's not equal?

```suggestion
      const needsUpdate = !isEqual(parsedBindParamsMap, prevQueryBindParams);
      if (!needsUpdate) {
        return prevQueryBindParams;
      }
```

Reply:
variable name changed!